kolibri/content/urls.py

```diff
@@ -201,34 +138,31 @@ def retrieve(self, request, pk=None, channelmetadata_channel_id=None):
         ).data
         return Response(file)
 
-    def update_content_copy(self, request, channelmetadata_channel_id, pk, content_copy, *args, **kwargs):
-        """
-        endpoint for content api method
-        update_content_copy(file_object=None, content_copy=None)
-        """
-        target_file = models.File.objects.using(channelmetadata_channel_id).get(pk=pk)
-        return Response(api.update_content_copy(file_object=target_file, content_copy=str(content_copy)))
+
+class FormatPresetViewset(viewsets.ViewSet):
+    def list(self, request, channelmetadata_channel_id=None):
+        context = {'request': request, 'channel_id': channelmetadata_channel_id}
+        files = serializers.FormatPresetSerializer(models.FormatPreset.objects.using(channelmetadata_channel_id).all(), context=context, many=True).data
```

Comment:
A bit confused by the name "files". Same below with `file`; these should probably be something like `presets` and `preset`, right?